Sources/TuistLoader/SwiftPackageManager/PackageInfoMapper.swift

```diff
@@ -542,7 +542,7 @@ public final class PackageInfoMapper: PackageInfoMapping {
                         .linker,
                         .define
                     ),
-                    (.linker, .unsafeFlags), (_, .enableExperimentalFeature):
+                    (.linker, .unsafeFlags), (_, .enableExperimentalFeature), (_, .swiftVersion):
```

Comment:
I don't fully understand this linker dependency stuff so this should be reviewed by someone.

Reply:
This is fine, you'd only return a non-nil value if the setting would add a new dependency on an SDK.